### DISCUSSION TITLE
Hide Cookie Consent on Privacy Page

### DISCUSSION THREAD

**Discussion by dombyte** [orestbida/cookieconsent#547] [Q&A] [answered]
I thought about hiding the Cookie Consent Pop Up on the Privacy Policy Page. 
My first idea was to just not load the JS but I want to be able to open the settings via an button but don't show pop up when visiting the Privacy page.
Does someone have any idea how this could be achieved?

**Comment by orestbida** [comment]
Hi @dombyte, here is an [example implementation](https://github.com/orestbida/cookieconsent/issues/287).

**Reply by dombyte** [reply]
Thanks, but it only works in the latest stable version. In version 3.x it won't work. Did v3 get rid of cc.autorun?

**Reply by dombyte** [reply, marked as answer]
I fixed it with using autoShow instead of autorun. Did this changed in v3?

**Reply by orestbida** [reply]
Yes, [v3's config and API is completely different from v2](https://cookieconsent.orestbida.com/essential/introduction.html#cookieconsent-v2-or-v3).
